src/pipeline/utils.py

```diff
@@ -1,11 +1,13 @@
 from huggingface_hub import HfApi
+from ..utils.import_utils import is_alignment_available
 
-from alignment import (
-    ModelArguments,
-    DataArguments,
-    H4ArgumentParser,
-    SFTConfig
-)
+if is_alignment_available():
```

Comment:
What is this for? 
I mean we specified alignments dependencies in requirements.txt

Reply:
Thanks for letting me know! 
Probably need to have three separate `requirements.txt` for each steps in the future.